replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -38,7 +39,8 @@
 
   private static final String SUSPENDED_KEY = "suspended";
 
-  public static final int CURRENT_VERSION = 3;
+  /** 1 - initial version. */
+  public static final int CURRENT_VERSION = 1;
```

Comment:
I've changed this back to 1 since it this will be the first version of this config that we save in the persistent store. Any previous version of config can be identified because it will be saved in the ddf catalog.

Reply:
I think this still needs to be 3 since it is the third version of the config. How it is persisted shouldn't really matter to this version.